styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -331,17 +364,156 @@
 
 .caseStudy-text {
   font-family: $copy;
-  font-size: 1.75rem;
+  font-size: 1.25rem;
   letter-spacing: .5px;
-  line-height: 2.5rem;
+  line-height: 1.5rem;
+  word-spacing: .5px;
```

Comment:
I know we're just following examples of code that was here already, but I'd like to avoid using pixels for word spacing.  For consistency, we should use `rem` for font sizes and then `em` for letter-spacing/word-spacing/etc....